src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,109 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int capacity;
+    private int threshold;
+
+    private Node<K, V>[] table;
+
     @Override
     public void put(K key, V value) {
-
+        table = resize();
```

Comment:
resize shoud be inside IF (we do not need resize each time when we add new element)